server/src/test/scala/io/delta/sharing/server/DeltaSharingServiceSuite.scala

```diff
@@ -1410,6 +1414,165 @@ class DeltaSharingServiceSuite extends FunSuite with BeforeAndAfterAll {
     verifyPreSignedUrl(actualFiles(1).url, 573)
   }
 
+  integrationTest("column mapping query test 1 - /shares/{share}/schemas/{schema}/tables/{table}/metadata") {
```

Comment:
could you update the test names to match the actual query you are using? `/metadata` -> `/query`

Reply:
fixed